### ISSUE TITLE
Unable to get any responses

### ISSUE THREAD

**Issue by Mpayne007** [disenone/UE.AIChatPlus.Public#2]
![image](https://github.com/user-attachments/assets/a40740a4-8a9b-4337-b336-18df06ea4b10)

Hello,
I am a Gemini AI user. I wanted to use this for help with blueprints. I can not get this to work at all. I have verified my API key is correct(removed for safety reasons) 

I am not sure what i am doing wrong exactly. 

Any help you can provide?

**Comment by disenone** [comment]
Hi, 
It seems you are getting empty response from Gemini,
Have you try any other tools to work with gemini?

**Comment by Mpayne007** [comment]
No i have not, this would be my frist app and first attempt to set up. Is there another application i can test with?

**Comment by disenone** [comment]
I would suggest Chatbox https://chatboxai.app/en

**Comment by Mpayne007** [comment]
Well, good news
![image](https://github.com/user-attachments/assets/c5d5fedc-cfc2-42f8-9281-5b8938b68216)
 
That worked like a charm. Its set up is super similiar to your plugin with unreal.

**Comment by disenone** [comment]
Hi，
I Found The Photo of Unreal you send has leaked you key of Gemini (The FullUrl Section)
Maybe you should Invalidate you key and create a new one

**Comment by Mpayne007** [comment]
thank you for looking out! i got it swapped.

**Comment by disenone** [comment]
Beside,
I still cannot get it Why Gemini does not work on my Plugin.
Could you please try create a new chat and send a message to Gemini in Unreal ?

**Comment by Mpayne007** [comment]
Yes, im also willing to get on a discord call if you wish at any point for you to troubleshoot.

**Comment by Mpayne007** [comment]
Made a new chat, sent test. still doesnt work. 
uninstalled and reinstalled plugin

**Comment by disenone** [comment]
Thanks a lot.
It is weird，I will try reproduce myself and maybe the loosen error check.

**Comment by Mpayne007** [comment]
So, i had a change in behavior, i went into the settings reset things, then applied my API key and chose the model, then the error changed
![image](https://github.com/user-attachments/assets/d3cc6f23-3b24-41fb-894d-ed4e104828f1)


I hope this helps, if there are logs please let me know.

I am going to install ollama and see if i can verify that works to ensure its not my system.

**Comment by disenone** [comment]
the error shows you have some empty message in you chat session
maybe you could delete the empty chat message

And could you try change setting Method from streamGenerateContent to generateContent ?

**Comment by Mpayne007** [comment]
updated the stream to generateContent instead. deleted chat, tried again with failure.

**Comment by disenone** [comment]
Fine, Thanks a lot.
I will try debugging the Plugin later.

**Comment by Mpayne007** [comment]
Thank you for taking time out of your day to help me. I hope you have a good one!
